internal/router/fieldinclude.go

```diff
@@ -38,6 +39,11 @@ func (u *FieldUpdater) UpdateSearch() {
 	u.search = true
 }
 
+// CopyACLs requests that the ACL records of entities are updated.
+func (u *FieldUpdater) CopyACLs() {
```

Comment:
UpdateACLs ?

After all, the copying of ACLs is really just an implementation artifact - the information we're conveying here is that the base ACLs have changed, I think.